imjoy/runners/subprocess.py

```diff
@@ -248,7 +248,8 @@ def logging_callback(msg, type="info"):  # pylint: disable=redefined-builtin
 
     token = kwargs.get("token")
     if token != engine.opt.token:
-        logger.debug("Token mismatch: %s != %s", token, engine.opt.token)
+        logger.error("Wrong connection token (%s)", token)
+        print("========>> Connection token: {} <<========".format(engine.opt.token))
```

Comment:
Use print to make sure we give the correct token no matter which log level it is.